src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -179,6 +179,7 @@ def load_arguments(self, _):
         c.argument('cluster_autoscaler_profile', nargs='+', options_list=["--cluster-autoscaler-profile", "--ca-profile"],
                    help="Space-separated list of key=value pairs for configuring cluster autoscaler. Pass an empty string to clear the profile.")
         c.argument('uptime_sla', action='store_true')
+        c.argument('tier', type=str)
```

Comment:
Also, I suppose this option should only accept 2 values `standard` and `free` for now, then we could set this as an enum type as it will place constraints on user input that we expect. May refer to `load_balancer_sku`.